### DISCUSSION TITLE
After upgrading the Go language to version 1.21

### DISCUSSION THREAD

**Discussion by softmgr** [ikemen-engine/Ikemen-GO#1951] [Other] [answered]
The latest development version of ikemen-go has upgraded the Go compiler to version 1.21, and now the minimum supported version for compiling and running is Windows 10. Will ikemen-go no longer support the Windows 7 operating system in the future?

**Comment by Kiwi1444** [comment]
And related question. Will I be able to compile Ikemen Go, using Go v1.20 in the future? If not, I will be forced to stop at the current nightly release to be able to launch Ikemen Go on my desktop with Windows 7.

**Comment by K4thos** [comment, marked as answer]
The latest nightly version already works on Windows 7, thanks to the kind efforts of @MarkKremer, the maintainer of beep. While we may eventually drop Windows 7 support, we plan to stay on Go 1.20 for as long as it remains feasible - until either core Go functionality makes it impractical or dependencies require an upgrade.
